src/main/fc/core.c

```diff
@@ -279,7 +279,7 @@ void updateArmingStatus(void)
 
 #ifdef USE_GPS_RESCUE
         if (gpsRescueIsConfigured()) {
-            if (!gpsRescueConfig()->minSats || STATE(GPS_FIX) || ARMING_FLAG(WAS_EVER_ARMED)) {
+            if (gpsRescueConfig()->allowArmingWithoutFix || STATE(GPS_FIX)) {
```

Comment:
One question: I have not too much knowledge about the GPS Rescue code, but I see you removed the `minSats` validation from here and I don't see it in other place. I suppose it must remain when `allowArmingWithoutFix` is `false`?

Reply:
True. I didn't see that the `minSats` here was only a boolean. Thanks for the explanation ;)